practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/model/Practitioner.java

```diff
@@ -1,12 +1,24 @@
 package uk.ac.ox.ndph.mts.practitioner_service.model;
 
-public class Practitioner {
+import com.fasterxml.jackson.annotation.JsonCreator;
 
+public class Practitioner {
+    private final String id;
     private final String prefix;
     private final String givenName;
     private final String familyName;
+    private String userAccountId;
+
+    @JsonCreator
+    public Practitioner(final String id, final String prefix, final String givenName, final String familyName) {
+        this.id = id;
+        this.prefix = prefix;
+        this.givenName = givenName;
+        this.familyName = familyName;
+    }
 
     public Practitioner(final String prefix, final String givenName, final String familyName) {
+        this.id = "";
```

Comment:
Bit of a dangerous constructor since the ID is not set, assume this is used in tests?

Reply:
Please look again and see if it's ok.